### PR TITLE
Updating names of tasks and their descriptions

### PR DESCRIPTION
**Pull request by maciej-majchrzak-wttech**
tasks

### CODE REVIEW

File: pkg/project/cloud/Taskfile.yml [hunk 1]
```diff
@@ -118,27 +121,29 @@ tasks:
...
-    desc: enable CRX/DE
+    desc: enables CRX/DE on all AEM instances
+    internal: true
     cmds:
       - 'sh aemw osgi config save --pid "org.apache.sling.jcr.davex.impl.servlets.SlingDavExServlet" --input-string "alias: /crx/server"'
 
   aem:provision:apm:
-    desc: deploy APM tool
+    desc: deploys APM tool
+    internal: true
```
Comment (krystian-panek-wttech) [inline pkg/project/cloud/Taskfile.yml:131]: I was thinking about it but for troubleshooting it worth sometimes to not have it internal to allow a standalone cmd execution?

File: pkg/project/cloud/Taskfile.yml [hunk 2]
```diff
@@ -118,27 +121,29 @@ tasks:
...
 
   aem:stop:
-    desc: stop AEM instances
+    desc: stops AEM instances
     aliases: [aem:down]
     cmds: [sh aemw instance stop]
 
   aem:build:
-    desc: build AEM application
+    desc: builds AEM application in current folder using Maven
```
Comment (krystian-panek-wttech) [inline pkg/project/cloud/Taskfile.yml:146]: what does it mean in current folder ?:D I don't get it why it is important to mention it here

File: pkg/project/cloud/Taskfile.yml [hunk 3]
```diff
@@ -147,31 +152,35 @@ tasks:
         --file "all/target/*.all-*.zip"
 
   aem:deploy:
-    desc: deploy AEM application
+    desc: deploys AEM application built with aem:build
```
Comment (krystian-panek-wttech) [inline pkg/project/cloud/Taskfile.yml:155]: it's too obvious for me

**Comment by krystian-panek-wttech** [comment]
provision desc is so loooong that the output is not looking so elegant...

there is also `--summary` option to put more text...

File: pkg/project/classic/Taskfile.yml [hunk 4]
```diff
@@ -12,77 +12,77 @@ vars:
...
+    desc: starts AEM instances
     aliases: [aem:up]
     cmds:
       - sh aemw instance create
       - sh aemw instance up
 
   aem:await:
-    desc: await stable AEM instances
+    desc: awaits stable AEM instances
+    internal: true
```
Comment (krystian-panek-wttech) [inline pkg/project/classic/Taskfile.yml:72]: this is sth useful when called standalone

Comment (maciej-majchrzak-wttech) [reply pkg/project/classic/Taskfile.yml:72]: Hmm - I'm not sure if we should expose these granular actions through tasks for other systems to use it. This whole Task tool was rather to combine actions...

**Comment by krystian-panek-wttech** [comment]
image

**Pull request merged**